src/handlers/payout/post.ts

```diff
@@ -86,29 +130,58 @@ export const incentivizeCreatorComment = async () => {
   }
   const context = getBotContext();
   const payload = context.payload as Payload;
-  const org = payload.organization?.login;
   const issue = payload.issue;
-  if (!issue || !org) {
-    logger.info(`Incomplete payload. issue: ${issue}, org: ${org}`);
+  if (!issue) {
+    logger.info(`Incomplete payload. issue: ${issue}`);
     return;
   }
-  const assignees = issue?.assignees ?? [];
+
+  if (issue.state_reason !== StateReason.COMPLETED) {
+    logger.info("incentivizeCreatorComment: comment incentives skipped because the issue was not closed as completed");
+    return;
+  }
+
+  if (paymentPermitMaxPrice == 0 || !paymentPermitMaxPrice) {
```

Comment:
Same here. `paymentPermitMaxPrice` is just being checked here for having some value, but it is not being enforced in this function. A big incentive is being generated https://github.com/web4erOrg/ubiquibot/issues/56